prow/cmd/gangway/main.go

```diff
@@ -147,33 +146,21 @@ func main() {
 		logrus.WithError(err).Fatal("unable to create prow job client")
 	}
 
-	// If we are provided credentials for Git hosts, use them. These credentials
-	// hold per-host information in them so it's safe to set them globally.
-	if o.cookiefilePath != "" {
-		cmd := exec.Command("git", "config", "--global", "http.cookiefile", o.cookiefilePath)
-		if err := cmd.Run(); err != nil {
-			logrus.WithError(err).Fatal("unable to set cookiefile")
-		}
-	}
-
-	gitClient, err := o.github.GitClientFactory(o.cookiefilePath, &o.config.InRepoConfigCacheDirBase, o.dryRun, false)
-	if err != nil {
-		logrus.WithError(err).Fatal("Error getting Git client.")
-	}
-
-	cacheGetter, err := config.NewInRepoConfigCache(o.config.InRepoConfigCacheSize, configAgent, gitClient)
-	if err != nil {
-		logrus.WithError(err).Fatal("Error creating InRepoConfigCacheGetter.")
-	}
 	metrics.ExposeMetrics("gangway", configAgent.Config().PushGateway, o.instrumentationOptions.MetricsPort)
 
 	// Start serving liveness endpoint /healthz.
 	healthHTTP := pjutil.NewHealthOnPort(o.instrumentationOptions.HealthPort)
 
+	var moonrakerClient *moonraker.Client
+	if o.moonrakerAddress == "" {
+		logrus.Fatal("moonrakerAddress cannot be empty")
+	}
```

Comment:
Hmm making moonraker mandatory for gangway may break existing gangway deployments, @jmguzik you're running gangway in openshift, right?